inc/saiswitch.h

```diff
@@ -1213,10 +1311,8 @@ typedef sai_status_t (*sai_get_switch_attribute_fn)(
  */
 typedef struct _sai_switch_api_t
```

Comment:
this is likely be in a separate CR. We need some example first, then we can change the ptf tests to use the new API.